### ISSUE TITLE
trivial solution for speaker overlap

### ISSUE THREAD

**Issue by danFromTelAviv** [pyannote/pyannote-audio#500]
**Describe the bug**
naturally, since there's very little time with speech overlap compared to regular speech or silence the model reaches a trivial solution of there is never any overlap. 

**To Reproduce**
```bash
$ pyannote-audio sad train ...
$ pyannote-audio sad validate ...
```

**Content of `config.yml`**

```yaml
task:
   name: OverlapDetection
   params:
      duration: 2.0
      batch_size: 64
      per_epoch: 1

feature_extraction:
   name: RawAudio
   params:
      sample_rate: 16000

architecture:
   name: pyannote.audio.models.PyanNet
   params:
      rnn:
         unit: LSTM
         hidden_size: 128
         num_layers: 2
         bidirectional: True
      ff:
         hidden_size: [128, 128]

scheduler:
   name: ConstantScheduler
   params:
      learning_rate: 0.01
```

**Additional context**
How did you get the overlap detector for AMI to train well? I suspect they don't have much more overlap than my data does. 
I feel like the right solution would be to use focal loss - but I can't find where you specify the loss in the code. 

I also see your loss reached around .2  - for me, it reaches around .01 or less.
